### DISCUSSION TITLE
Viewing a CSV file - last row covered by scrollbar

### DISCUSSION THREAD

**Discussion by patrickhlauke** [community/community#49077] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When viewing a CSV file, which is nicely rendered as an HTML table ... the last row of the table - if it needs horizontal scrolling - is partially covered by the scrollbar in Windows/Chrome

![screenshot showing the bottom of a wide table, with the horizontal scrollbar covering half of the last row](https://user-images.githubusercontent.com/895831/222706852-d5014741-a445-47f6-9d36-f9a3c1942f76.png)

The solution is likely the same as/similar to this proposed tweak on Bootstrap https://github.com/twbs/bootstrap/pull/37694#issuecomment-1383224723

**Comment by patrickhlauke** [comment]
seems to actually be related to the explicit `height: …` given to the table... disabling this solves the issue

![screenshot of the same table, with chrome developer tools visible. disabling the `height: 1190px` CSS element style fixes the scrollbar, making it not overlap the last row](https://user-images.githubusercontent.com/895831/222740946-ac463056-3e95-428f-858b-6c888593bd84.png)

**Comment by wkumler** [comment]
Confirming that this is still a problem (at least in Chrome) and can be fixed by disabling the explicit height parameter. Would be nice to see this implemented!

**Comment by patrickhlauke** [comment]
strangely, i can't seem to replicate the issue anymore (though wondering if i've now opted into some experiment/new view that fixes it)

**Reply by patrickhlauke** [reply]
even on really long and wide CSVs (@wkumler i took one of yours https://github.com/wkumler/MS_metrics/blob/491deaf1d5f27f9d276e58acb4c1dfca2a2e21b9/MW_upload/clean_stans.csv as an example), while the last row is still partially covered by the horizontal scrollbar, at least now there's also a vertical scrollbar that lets you scroll down to reveal it fully. that was originally missing altogether.

![screenshot of the last row in a large CSV - the last row is covered by the scrollbar, but there's a vertical scrollbar present to bring it fully into view](https://github.com/community/community/assets/895831/225c1a73-869c-498b-a752-6422eb642dea)

**Reply by patrickhlauke** [reply]
however, if the height CSS property is removed, this extra vertical scrollbar could be avoided altogether
